api/base/settings/defaults.py

```diff
@@ -51,21 +51,24 @@
 }
 
 REST_FRAMEWORK = {
-    'PAGE_SIZE': 10,
-
-    # Order is important here because of a bug in rest_framework_swagger. For now,
```

Comment:
Yes, that brings swagger docs back, but then the browsable API is using the 'rest_framework.renderers.JSONRenderer' as well, and there is no formatting.  Should I use custom content negotiation?